openverse_catalog/dags/common/requester.py

```diff
@@ -90,10 +93,17 @@ def get(self, url, params=None, **kwargs):
         except Exception as e:
             logger.error(f"Error with the request for URL: {url}")
             logger.info(f"{type(e).__name__}: {e}")
-            logger.info(f"Using query parameters {params}")
+            if params := request_kwargs.get("params"):
+                logger.info(f"Using query parameters {params}")
             logger.info(f'Using headers {request_kwargs.get("headers")}')
             return None
 
+    def get(self, url, params=None, **kwargs):
+        self._make_request(self.session.get, url, params=params, **kwargs)
+
+    def head(self, url, **kwargs):
+        self._make_request(self.session.head, url, **kwargs)
```

Comment:
Oh dang, I'm glad I tested this! these 2 functions need to return the result of `_make_request` so that the callers of each can process the result of the request. My local runs were failing until I added the `return`s:
```suggestion
    def get(self, url, params=None, **kwargs):
        return self._make_request(self.session.get, url, params=params, **kwargs)

    def head(self, url, **kwargs):
        return self._make_request(self.session.head, url, **kwargs)
```

Reply:
Oh wow. 😅 Thanks a lot for the testing tests [you provided](https://github.com/WordPress/openverse-catalog/pull/865#pullrequestreview-1183591118)!